gradle/spec/fixtures/buildfiles/lockfile/app/src/main/java/org/example/App.java

```diff
@@ -0,0 +1,14 @@
+/*
```

Comment:
Probably we can trim some of these files, but I wasn't sure which ones and there's something to be said for having a complete example.

Reply:
I think everything except `build.gradle.kts` and `gradle.lockfile` are unnecessary.

Reply:
Given that this PR relies on running the `build` task, you either need to preserve the configuration cache option or a single source file per source set. Otherwise dependencies will not be resolved as Gradle tasks would not need to run.

Reply:
Do I understand correctly that that means we can delete the following:

1. gradle.properties
2. gradle\
3. gradlew
4. gradlew.bat

Reply:
Unless you run a command in an environment where Gradle is installed, you need those for[ Gradle to bootstrap itself](https://docs.gradle.org/current/userguide/gradle_wrapper.html).

Reply:
Ah, then we should keep everything that's already here?